### DISCUSSION TITLE
GitHub signup page is slow because of stars animation

### DISCUSSION THREAD

**Discussion by bert-w** [community/community#27287] [Other Feature Feedback, Questions, & Ideas]
https://github.com/signup

This page is really slow and laggy when I run it on Firefox (Windows). It runs fine on Chrome. This is probably because of a big difference in render engine.

**Comment by jfullstackdev** [comment]
yup and if that is the case, maybe we can consider it not on the GitHub side but rather on the browser side,

**Reply by bert-w** [reply]
I'm not sure what you mean; it's clientside code that runs?

**Comment by AbdullohRazzoqov** [comment]
hi

**Reply by iLikepackagefilesnzips** [reply]
hey
